examples/js/controls/OrbitControls.js

```diff
@@ -255,8 +255,8 @@ THREE.OrbitControls = function ( object, domElement ) {
 		scope.domElement.removeEventListener( 'touchend', onTouchEnd, false );
 		scope.domElement.removeEventListener( 'touchmove', onTouchMove, false );
 
-		document.removeEventListener( 'mousemove', onMouseMove, false );
-		document.removeEventListener( 'mouseup', onMouseUp, false );
+		scope.domElement.removeEventListener( 'mousemove', onMouseMove, false );
```

Comment:
This change will break applications where the canvas is not full screen. As you can see at the following fiddle, it's not possible to transform the camera when you leave the canvas with the mouse.

https://jsfiddle.net/wfc48ukz/

However, it should be like so:

https://jsfiddle.net/wfc48ukz/1/

Reply:
On my PC (Win 10) and on the newest Firefox, Chrome and Edge the fiddles work exactly the same?
I'm sorry for the removal of the document usage.